html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="email" id="email" name="email" />
+
+      <div>
+        <label for="select">Why is coding cool?</label>
+      </div>
+      <select id="select" name="Why is Coding Cool?">
+        <option value="n/a">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+
+      <div>Favorite Language (choose one)</div>
+      <input type="radio" id="language1" name="coding" value="language1" />
+      <label for="language1">HTML</label>
+      <input type="radio" id="language2" name="coding" value="language2" />
+      <label for="language2">CSS</label>
+      <input type="radio" id="language3" name="coding" value="language3" />
```

Comment:
this works, but I'd recommend setting the value to `HTML` or `CSS` so it makes more sense. Whatever is stored in the `value` attribute is what will be passed to the server when the form is submitted. If you're storing the value of `langauge1` to the server, it'll be less human readable later when you need to process that information.